src/plugins/irc-events/link.js

```diff
@@ -23,14 +23,19 @@ module.exports = function(client, chan, msg) {
 		return;
 	}
 
-	const link = escapeHeader(links[0].link);
-	fetch(link, function(res) {
-		if (res === null) {
-			return;
-		}
+	Array.from(new Set( // Remove duplicate links
```

Comment:
I agree with the intent, but this goes way beyond the goal of this PR and the time I can allocate on this. I would rather keep the semi-naive solution I have right now (which is not worse than what we have in `master` anyway), and we can work on proper caching/deduping in a later PR, IMO.